ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/activity/PaywallActivityArgs.kt

```diff
@@ -1,9 +1,17 @@
 package com.revenuecat.purchases.ui.revenuecatui.activity
 
 import android.os.Parcelable
+import com.revenuecat.purchases.ui.revenuecatui.fonts.FontResourceProvider
+import com.revenuecat.purchases.ui.revenuecatui.fonts.TypographyType
 import kotlinx.parcelize.Parcelize
 
 @Parcelize
 internal data class PaywallActivityArgs(
     val offeringId: String? = null,
-) : Parcelable
+    val fonts: Map<TypographyType, Int?>? = null,
+) : Parcelable {
+    constructor(offeringId: String? = null, provider: FontResourceProvider?) : this(
```

Comment:
Maybe?

```suggestion
    constructor(offeringId: String? = null, fontProvider: FontResourceProvider?) : this(
```

So it's clear what it provides